mindsdb/api/executor/command_executor.py

```diff
@@ -1095,8 +1095,8 @@ def _create_integration(self, name: str, engine: str, connection_args: dict):
             raise EntityExistsError('Project exists with this name', name)
 
         self.session.integration_controller.add(name, engine, connection_args)
+        handler = self.session.integration_controller.get_data_handler(name, connect=False)
```

Comment:
we can move this line back under 'if storage:' condition. it will speed up a little - hanlder will be created only when it is going to be updated by import

Reply:
Oh, yes. It's done!